src/app/lightning/delete-ln-payments.ts

```diff
@@ -46,7 +47,22 @@ const checkAndDeletePaymentForHash = async ({
     },
     async () => {
       const lnPayment = await LnPaymentsRepository().findByPaymentHash(paymentHash)
-      if (lnPayment instanceof Error) return lnPayment
+      if (lnPayment instanceof Error) {
+        if (lnPayment instanceof CouldNotFindLnPaymentFromHashError) {
+          const lnd = LndService()
+          if (lnd instanceof Error) return lnd
+          const lnPaymentLookup = await lnd.lookupPayment({ pubkey, paymentHash })
+          if (lnPaymentLookup instanceof Error) return lnPaymentLookup
+
+          LnPaymentsRepository().persistNew({
```

Comment:
this will be executed from cron, we need to make it sync

Reply:
Done